index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="portfolio.css">
+    <title>Nsanzimfura portfolio</title>
+
+</head>
+<body>
+    <header>
+        <nav class="top-navigation-bar">
+            <h1 class="header-welcome">Welcome</h1>
+        </nav>
+        <img class="menu-icon" src="./assets/header-files/icon-left-side-menu.svg" alt="hamburger-icon" />
+    </header>
+    <main>
+        <section class="main-page">
+            <h3 class="introduction-text"><span class="dark-blue">Hey there. I'm <br>Eden<br></span>
+            <span class="green-content">I'm a software developer</span></h3>
+            <p class="description-profession">
+                I can help you build a product , feature or website Look through some
+                of my work and experience! If you like what you see and have a project
+                you need coded, don’t hesitate to contact me.</p>
+
```

Comment:
You did a good job here, but as you know it's very important to follow the exact Figma design, especially the fonts, colors, and sizing. So please make sure to update your font and font size for your header.

![image](https://user-images.githubusercontent.com/34779928/198399026-3128405e-5e29-4f0f-9cc9-82185bad7c75.png)

Reply:
- [ ] Please to continue with your project, kindly add the changes requested for the previous reviewer 🥇

Reply:
**OPTIONAL** - Please fix the responsiveness of your navbar 🙏 

Your project:
<img width="667" alt="navbar today" src="https://user-images.githubusercontent.com/73607512/198563449-21dc848c-1556-4c42-9718-45964dc5f155.PNG">